### PR TITLE
Modified README.md

### PR DESCRIPTION
**Pull request by xma8lu**
Please Merge 
#Hacktoberfest

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,11 +43,11 @@ foreman run java -cp build/libs/vonage-java-code-snippets-with-dependencies.jar
 If you set the environment variable `QUICKSTART_DEBUG` to any value, extra information
 will be output to the console from the Vonage Server SDK.
 
-## Request an Example
+## 🙏 Request an Example
```
Comment (yallen011) [inline README.md:46]: Can you change the emoji to a different emoji?

File: README.md [hunk 2]
```diff
@@ -43,11 +43,11 @@ foreman run java -cp build/libs/vonage-java-code-snippets-with-dependencies.jar
...
 
 Please [raise an issue](https://github.com/nexmo-community/nexmo-java-quickstart/issues) to request an example that isn't present within the quickstart. Pull requests will be gratefully received.
 
-## License
+## 📝 License
```
Comment (yallen011) [inline README.md:50]: Can you change the emoji to a different emoji?

**Review by yallen011 — CHANGES_REQUESTED** [review]

**Comment by yallen011** [comment]
Hi @aditya-gite-04 where did you get these emojis?

**Comment by aditya-gite-04** [comment]
Hey,
You can get all the usable emojis
https://github.com/ikatyang/emoji-cheat-sheet

File: README.md [hunk 3]
```diff
@@ -43,11 +43,11 @@ foreman run java -cp build/libs/vonage-java-code-snippets-with-dependencies.jar
 If you set the environment variable `QUICKSTART_DEBUG` to any value, extra information
 will be output to the console from the Vonage Server SDK.
 
-## 🙏 Request an Example
+## :octocat: Request an Example
```
Comment (yallen011) [inline README.md:46]: Can you change the :octocat: to a 🛎️  then I you will be all set to merge.

**Review by yallen011 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (aditya-gite-04) [reply README.md]: done

**Pull request closed without merging**